### ISSUE TITLE
Add auto closing tag for XAML

### ISSUE THREAD

**Issue by gubhit1s** [microsoft/vscode-dotnettools#1239]
### Describe the feature you'd like

 I already saw the intellisense for XAML which is great but I want close tag for XAML just like XML, or like when I use Visual Studio 2022. Is that possible?

For example, when I write
`<VerticalStacklayout`
then when adding the close bracket, it should also add the closing tag, then pressing enter should move the cursor like this:
```
<VerticalStacklayout>
   <!-- cursor is here -->
</VerticalStacklayout>
```

### Environment Information

Vs Code: 1.90.2 - Mac OS
C# dev kit: 1.7.27
MAUI: 1.0.6

**Comment by mgoertz-msft** [comment]
Auto-insert of end tag when typing `>` should already work. With the latest C# Extension I get this on Windows (haven't tried MacOS):

`<VerticalStacklayout|` + `>` => `<VerticalStacklayout>|</VerticalStacklayout>`

Pressing `Enter` produces this though:

`<VerticalStacklayout>`
`|</VerticalStacklayout>`

Getting it to insert a new indented line that would be a new feature request.

**Comment by mgoertz-msft** [comment]
@gubhit1s This is available in `prerelease` now. To try the fix please update your C# extension to `v2.43.16 (pre-release)`.

```xaml
<Button>|</Button>
=>
<Button>
    |
</Button>
```

and

```xaml
<Button Width="100" |
=>
<Button Width="100"
    |
```

Everything else will get the previous indent as before.
